docs/intro/contributing.txt

```diff
@@ -398,7 +398,7 @@ function to the correct file.
 Running Django's test suite for the second time
 ===============================================
 
-Once you've verified that your patch and your test are working correctly, it's
+Once you've verified that your chnages and your test are working correctly, it's
```

Comment:
Typo:
```suggestion
Once you've verified that your changes and your test are working correctly, it's
```

Reply:
@felixxm addressed in c525981b267f6fd7611d300e2df7666ca7d9ca4b